### ISSUE TITLE
Client-side form validation : aria-invalid on label

### ISSUE THREAD

**Issue by vinespie** [joomla/joomla-cms#6596]
<h2>Client-side form validation : aria-invalid on label</h2>

Hello,

When using the client-side form validation system joomla " behavior.formvalidator " when a field is misinformed and when the field is linked to a label, the attribute " aria -invalid = true" is added to the label.
According to accessibility standards this attribute should be only placed on the attribute "input".

Thank you for your answers

http://www.w3.org/TR/wai-aria/states_and_properties#aria-invalid
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-invalid_attribute

<h3>Example before client-side validation :</h3>
```html
<form  class="form-validate" method="post">
     <label id="jform_title-lbl" title="" for="jform_title">Title</label>
     <input id="jform_title" type="text" aria-required="true" required="required">
</form>
```
<h3>Example after client-side validation wrong :</h3>
```html
<form  class="form-validate" method="post">
     <label id="jform_title-lbl" title="" for="jform_title" aria-invalid="true">Title</label>
     <input id="jform_title" type="text" aria-required="true" required="required" aria-invalid="true">
</form>
```
<h3>Example after client-side validation correct:</h3>
```html
<form  class="form-validate" method="post">
     <label id="jform_title-lbl" title="" for="jform_title">Title</label>
     <input id="jform_title" type="text" aria-required="true" required="required" aria-invalid="true">
</form>
```

<h3>Patch to \media\system\js\validate-uncompressed.js</h3>
Remove line 52
```javascript
if($label){
     $label.addClass('invalid');
}
```
Remove line 57
```javascript
if($label){
     $label.removeClass('invalid');
}
```

**Comment by bertmert** [comment]
If you're right (I don't know but think so after reading also https://www.w3.org/WAI/GL/wiki/Using_Aria-Invalid_to_Indicate_An_Error_Field#Example_1)

your patch should be

Change line 52
```
if($label){
     $label.addClass('invalid');
}
```

Change line 57
```
if($label){
     $label.removeClass('invalid');
}
```

**Comment by vinespie** [comment]
Yes @bertmert , leave the class "invalid" just delete "aria- invalid"

**Comment by zero-24** [comment]
Closing and copy the description as we have a PR (https://github.com/joomla/joomla-cms/issues/6613) thanks!
